test/e2e/main_test.go

```diff
@@ -98,23 +96,24 @@ type InstallTestCase struct {
 
 func TestMeteringUpgrades(t *testing.T) {
 	tt := []struct {
-		Name                      string
-		MeteringOperatorImageRepo string
-		MeteringOperatorImageTag  string
-		Skip                      bool
-		PurgeReports              bool
-		PurgeReportDataSources    bool
-		ExpectInstallErr          bool
-		ExpectInstallErrMsg       []string
-		InstallSubTest            InstallTestCase
-		MeteringConfigSpec        metering.MeteringConfigSpec
+		Name                           string
+		MeteringOperatorImageRepo      string
+		MeteringOperatorImageTag       string
+		Skip                           bool
+		PurgeReports                   bool
+		PurgeReportDataSources         bool
+		ExpectInstallErr               bool
+		ExpectInstallErrMsg            []string
+		InstallSubTest                 InstallTestCase
+		MeteringConfigManifestFilename string
 	}{
```

Comment:
Nit: indents on the values

Reply:
All of that is automatically controlled by `gofmt`. It's part of the `make verify` check as well.